clang/Maintainers.rst

```diff
@@ -69,6 +69,12 @@ Sema
 | aeternalmail\@gmail.com (email), Sirraide (GitHub), Ætérnal (Discord), Sirraide (Discourse)
 
 
+Recovery AST
+~~~~~~~~~~~~
+| Haojian Wu
+| hokein.wu\@gmail.com (email), hokein (Phabricator), hokein (GitHub)
```

Comment:
If you have a Discord or Discourse handle, it'd be nice to list it.

Reply:
Added Discourse handle.